mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -2486,7 +2616,7 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.AuthenticatorGroupsConfig = expandAuthenticatorGroupsConfig(v)
 	}
 
-	if v, ok := d.GetOk("private_cluster_config"); ok {
+	if v, ok := d.GetOk("private_cluster_config.0.master_ipv4_cidr_block"); ok {
```

Comment:
We need to call this with the same value as before, or modify the expand function to expect the new value- we're currently running into an issue because the types have changed:

> panic: interface conversion: interface {} is string, not []interface {}
> 
> goroutine 22369 [running]:
> github.com/hashicorp/terraform-provider-google-beta/google-beta/services/container.expandPrivateClusterConfig(...)
> 	/go/src/github.com/modular-magician/terraform-provider-google-beta/google-beta/services/container/resource_container_cluster.go:5335

Exact line #s in this case correspond to the diff at https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-12025-old..auto-pr-12025, and will be erased on a push. L5536 is the corresponding line in this file.